api/handler/acl_test.go

```diff
@@ -372,15 +372,15 @@ func TestAstToTable(t *testing.T) {
 
 	expectedTable := eacl.NewTable()
 	record := eacl.NewRecord()
-	record.SetAction(eacl.ActionAllow)
-	record.SetOperation(eacl.OperationPut)
-	eacl.AddFormedTarget(record, eacl.RoleUser, *(*ecdsa.PublicKey)(key.PublicKey()))
+	record.SetAction(eacl.ActionDeny)
+	record.SetOperation(eacl.OperationGet)
+	eacl.AddFormedTarget(record, eacl.RoleOthers)
+	record.AddObjectAttributeFilter(eacl.MatchStringEqual, object.AttributeFileName, "objectName")
 	expectedTable.AddRecord(record)
 	record2 := eacl.NewRecord()
-	record2.SetAction(eacl.ActionDeny)
-	record2.SetOperation(eacl.OperationGet)
-	eacl.AddFormedTarget(record2, eacl.RoleOthers)
-	record2.AddObjectAttributeFilter(eacl.MatchStringEqual, object.AttributeFileName, "objectName")
+	record2.SetAction(eacl.ActionAllow)
+	record2.SetOperation(eacl.OperationPut)
+	eacl.AddFormedTarget(record2, eacl.RoleUser, *(*ecdsa.PublicKey)(key.PublicKey()))
```

Comment:
After https://github.com/nspcc-dev/neofs-s3-gw/pull/581 we shouldn't user `eacl.RoleUser` along with keys